lib/ansible/modules/network/nxos/nxos_lldp_global.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for nxos_lldp_global
+"""
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+DOCUMENTATION = """
+---
+module: nxos_lldp_global
+version_added: 2.9
+short_description: Configure and manage Link Layer Discovery Protocol(LLDP) attributes on NX-OS platforms.
+description: This module configures and manages the Link Layer Discovery Protocol(LLDP) attributes on NX-OS platforms.
+author: Adharsh Srivats Rangarajan (@adharshsrivatsr)
+notes:
+  - Tested against <network_os> 7.3.(0)D1(1) on VIRL
```

Comment:
In the notes, Add that `feature lldp` has to be enabled in order to use this module.